### ISSUE TITLE
Tanning takes a REALLY long time & other campfire issues

### ISSUE THREAD

**Issue by unmoving-javi** [TerraformersMC/Campanion#129]
I'm sure you've heard this one before, but tanning takes a long time. I get it, it takes a long time in real life too, but even with the campfire I'm only getting one or two pieces per ingame day. Given that you need five for a single big pouch, and 3 big pouches for a tent bag... well, you get the idea.

Now, I'm not saying that the time it takes to tan things needs to be reduced, but it would be nice to have some more visual feedback so you can tell approximately how much longer it'll take (maybe have the leather darken over time). This would make things a little bit less annoying, in my opinion anyways.

Also, I've put 3 tanning racks directly adjacent to a campfire (in each cardinal direction, facing out), and only 1 seems to be getting the speed bonus. Is that a bug, or is that intentional?

**Comment by Dragnier** [comment]
Yes, it looks like only one tanning rack is getting the bonus per fire for me too. I also agree that a gradual change of color would be helpful and that even with the fire assisting it takes far too long to make for a game.
